pkgs/applications/misc/openrgb/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv
+, qmake
+, fetchFromGitHub
+, fetchFromGitLab
+, libusb1 
+, hidapi
+, qt5
+, wrapQtAppsHook
+, makeWrapper
+, hicolor-icon-theme
+, openrazer-daemon
+}:
+stdenv.mkDerivation rec {
+  pname = "openrgb";
+  version = "unstable-${date}";
+
+  date = "2020-03-03";
+
+  src = fetchFromGitHub {
```

Comment:
Used GitHub because we need submodules, and the source seems to be equivalent with what's available on GitLab